security/pkg/pki/util/keycertbundle.go

```diff
@@ -233,6 +233,24 @@ func (b *KeyCertBundleImpl) ExtractCACertExpiryTimestamp() (float64, error) {
 	return extractCertExpiryTimestamp("CA cert", b.GetCertChainPem())
 }
 
+// TimeBeforeNextRotate returns time duration to the next cert rotation
+func TimeBeforeNextRotate(certBytes []byte, workloadTTL time.Duration, now time.Time) (time.Duration, error) {
```

Comment:
Do we need the now as an argument? why not call it inside the function.

Reply:
now is passed as a parameter to make it easier to test, I'll change since I believe there is no need for that.